packages/perseus/src/widgets/explanation.tsx

```diff
@@ -110,53 +111,18 @@ class Explanation extends React.Component<Props, State> {
                 </div>
             );
         } else {
-            if (isArticle) {
-                // NOTE: For articles, the baseElements `Link` may is either be
-                // an anchor tag (if rendered in a modal) or a Wonder Blocks
-                // link component in lesson page, see: `article-page.jsx`.
-                linkContainer = (
-                    <div
-                        className={css(
-                            styles.linkContainer,
-                            styles.articleLink,
-                        )}
-                    >
-                        <Link
-                            className={css(styles.explanationLink)}
-                            // HACK(michaelpolyak): WB Link doesn't passthrough
-                            // class name.
-                            style={styles.explanationLink}
-                            href={href}
-                            onClick={onClick}
-                            role="button"
-                            aria-expanded={this.state.expanded}
-                        >
-                            {`[${linkAnchor}]`}
-                        </Link>
-                    </div>
-                );
-            } else {
-                // NOTE: For exercises, the baseElements `Link` is an
-                // anchor tag, see: `perseus-api.jsx`.
-                linkContainer = (
-                    <div
-                        className={css(
-                            styles.linkContainer,
-                            styles.exerciseLink,
-                        )}
-                    >
-                        <Link
-                            className={css(styles.explanationLink)}
-                            href={href}
-                            onClick={onClick}
-                            role="button"
-                            aria-expanded={this.state.expanded}
-                        >
-                            {`[${linkAnchor}]`}
-                        </Link>
-                    </div>
-                );
-            }
+            const viewStyling = [styles.linkContainer, styles.explanationLink];
+            viewStyling.push(
+                isArticle ? styles.articleLink : styles.exerciseLink,
+            );
```

Comment:
Could this style be defined inline?
```suggestion
            const viewStyling = [
                styles.linkContainer,
                styles.explanationLink,
                isArticle ? styles.articleLink : styles.exerciseLink,
            ];
```
In general, our style is to avoid mutating arrays and POJOs where possible (though the mutation here is pretty harmless). TypeScript's type checking is less robust in the presence of mutability, because TS doesn't have a concept of [variance](https://flow.org/en/docs/lang/variance/).